python/src/StationaryFunctionalCovarianceModel_doc.i.in

```diff
@@ -38,11 +38,41 @@ CovarianceModel
 
 Examples
 --------
-Create a covariance model defined by a damped cosine correlation function:
+Create a covariance model defined by a damped cosine correlation function as follows:
+
+.. math::
+
+    \rho(\tau) = e^{-\tau}\cos \left( 2\pi\tau \right)
 
 >>> import openturns as ot
 >>> rho = ot.SymbolicFunction(['tau'], ['exp(-tau)*cos(2*pi_*tau)'])
 >>> covModel = ot.StationaryFunctionalCovarianceModel([1.0], [1.0], rho)
 >>> tau = [0.1]
 >>> print(covModel(tau))
-[[ 0.732029 ]]"
+[[ 0.732029 ]]
+
+If the correlation function has hyperparameters, we must turn it into a ParametricFunction.
+In the example below, we illustrate this by introducing a power parameter :math:`n` to the dampening function:
+
+.. math::
+
+    \rho(\tau) &= e^{-\tau}\left(\cos \left( 2\pi\tau \right)\right)^n \\
+               &= e^{-\tau}e^{n\times \ln\left(\cos \left( 2\pi\tau \right)\right)}
+
+>>> rho_param = ot.SymbolicFunction(['tau','n'], ['exp(-tau)*exp(n * log(cos(2*pi_*tau)))'])
+>>> rho = ot.ParametricFunction(rho_param, [1], [2.0])
+>>> covModel = ot.StationaryFunctionalCovarianceModel([1.0], [1.0], rho)
+
+Active parameters of a :class:`~openturns.CovarianceModel` are those that must be estimated.
+By default, only scale (here parameter number #0) and amplitude (here parameter #1) are active.
```

Comment:
the amplitude parameter index depends on the input dimension

Reply:
Yes, that is why it is stated to be parameter # 1 "here": "(here parameter # 1)". Do you think it should be more explicit?

Reply:
I think that the sentence is clear, but an extra comment would improve it. Suggestion : "More generally, if the covariance model has input dimension d, the scale parameters have indices from 0 to d-1 and the amplitude parameter has index d."

Reply:
This will no longer be true after we introduce the `IsotropicCovarianceModel` class.

Reply:
I do not understand: why should a new covariance model change the order of the parameters?

Reply:
An `IsotropicCovarianceModel` does not change the order of the parameters, but the number of parameters.

In an `IsotropicCovarianceModel` there is only 1 scale parameter regardless of the input dimension because all computations are handled by the underlying `CovarianceModel` with input dimension 1.

Reply:
https://github.com/openturns/openturns/blob/1dc2d083d5389cfccceb00e165e363237d828f92/lib/src/Base/Stat/IsotropicCovarianceModel.cxx#L136

Reply:
Marking this conversation as resolved because I have pushed a change that makes everything explicit.